### PR TITLE
build base class and construct sample response

### PR DESCRIPTION
**Pull request by mingkang111**
# Description
This PR aims to setup handler for evaluation. It also provides sample response for apis. Actual implementation will be in another PR after this.

Please checkout confluence page for more details.

 # Test Supported APIs in this PR
```
POST http://localhost:8888/aqua/evaluation/
GET http://localhost:8888/aqua/evaluation/
GET http://localhost:8888/aqua/evaluation/ocid1.datasciencemodel.<OCID>
GET http://localhost:8888/aqua/evaluation/ocid1.datasciencemodel.<OCID>/metrics
GET http://localhost:8888/aqua/evaluation/ocid1.datasciencemodel.<OCID>/report
GET http://localhost:8888/aqua/experiment/
DELETE http://localhost:8888/aqua/evaluation/ocid1.datasciencemodel.oc1.iad.<OCID>
PUT http://localhost:8888/aqua/evaluation/ocid1.datasciencemodel.oc1.iad.<OCID>/cancel
```

### CODE REVIEW

File: ads/aqua/evaluation.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+from ads.aqua import logger
+from ads.aqua.base import AquaApp
+from ads.common import oci_client as oc
+from ads.common.oci_resource import SEARCH_TYPE, OCIResource
+from ads.common.serializer import DataClassSerializable
+from ads.common.utils import get_console_link
+
+
+@dataclass(repr=False)
+class AquaEval(DataClassSerializable):
```
Comment (mrDzurb) [inline ads/aqua/evaluation.py:22]: Let's to be consistent and use Evaluation instead Eval everywhere?
I think something like ```ResourceIdentifier``` would be a better name for this class.

File: ads/aqua/evaluation.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+class AquaEvaluationSummary(DataClassSerializable):
+    """Represents a summary of Aqua evalution."""
+
+    id: str
+    name: str
+    console_url: str
+    lifecycle_state: str
+    lifecycle_details: str
+    time_created: str
+    experiment: AquaEval
```
Comment (mrDzurb) [inline ads/aqua/evaluation.py:55]: Let's use ```field(default_factory=ResourceIdentifier)```

File: ads/aqua/evaluation.py [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    source: AquaEval
+    job: AquaEval
+    tags: dict
+
+
+@dataclass(repr=False)
+class AquaEvaluationDetails(AquaEvaluationSummary):
+    """Represents a detail of Aqua evalution."""
+
+    parameters: str
```
Comment (mrDzurb) [inline ads/aqua/evaluation.py:65]: Double check types

Comment (mingkang111) [reply ads/aqua/evaluation.py]: Thanks for the reminder. Fixed.

File: ads/aqua/evaluation.py (hunk 2 above)
Comment (mingkang111) [reply ads/aqua/evaluation.py]: The implementation now is for getting mock response. Will take care of it in the real implementation. Thanks for the suggestion!.

File: ads/aqua/evaluation.py (hunk 1 above)
Comment (mingkang111) [reply ads/aqua/evaluation.py]: Thanks for the suggestion. Fixed the naming.

**Review by VipulMascarenhas — APPROVED** [review]

**Review by mrDzurb — APPROVED** [review]

**Pull request merged**